src/test/func/malloc/malloc.cc

```diff
@@ -15,23 +17,23 @@ void check_result(size_t size, size_t align, void* p, int err, bool null)
   bool failed = false;
   if (errno != err && err != SUCCESS)
   {
-    printf("Expected error: %d but got %d\n", err, errno);
+    std::cout << "Expected error: " << err << " but got " << errno << std::endl;
```

Comment:
These have all been replaced with the INFO and EXPECT macros.  If there are any other places where they're necessary then please can you use the new formatting infrastructure rather than iostream?